### ISSUE TITLE
重建 slack.com：使用 CSS Grid 重新设计，并针对性能和可访问性进行了优化。

### ISSUE THREAD

**Issue by leviding** [xitu/gold-miner#2416]
* 原文链接：[Rebuilding slack.com: A redesign powered by CSS Grid and optimized for performance and accessibility.](https://slack.engineering/rebuilding-slack-com-b124c405c193)
  * Markdown文件：[文件地址](https://github.com/xitu/gold-miner/blob/master/TODO/rebuilding-slack-com.md)
  * PR 地址：#2422
  * 文章分类：**前端**
  ----
  * 翻译时间：4 天
  * 校对时间：1 天
  * 翻译积分：4 分
  * 校对积分：1 分
  * [积分有什么用](https://github.com/xitu/gold-miner/wiki)

**Comment by Raoul1996** [comment]
翻译认领

**Comment by fanyijihua** [comment]
@Raoul1996 棒极啦 :tada:

**Comment by fanyijihua** [comment]
PR 地址：#2422
